data/json/items/vehicle/engine.json

```diff
@@ -344,6 +344,8 @@
     "name": { "str": "25 L 650 HP gas turbine engine" },
     "description": "A gas turbine engine.  Used primarily for small helicopters.",
     "//": "Based on Allison 250-C30R.  Doubled the published weight to account for a transmission.",
+    "//2": "It's from 60s, tempered steel might be appropriate?"
```

Comment:
It doesn't need to match the specs of exact item the stats are derived for, but if they are generally made out of tempered steel, yes, it makes total sense to use tempered steel material